Doc/library/shutil.rst

```diff
@@ -51,7 +51,8 @@ Directory and files operations
 .. function:: copyfile(src, dst, *, follow_symlinks=True)
 
    Copy the contents (no metadata) of the file named *src* to a file named
-   *dst* and return *dst*.  *src* and *dst* are path names given as strings.
+   *dst* and return *dst*.  *src* and *dst* are path names given as strings
+   in the most efficient way possible.
```

Comment:
This new doc snippet looks like it should be in the first sentence of this paragraph, not the second - it's the file copy that is performed in the most efficient way, not passing the path names in as strings.

Reply:
fixed